packages/quantic/force-app/main/default/lwc/quanticSummary/quanticSummary.js

```diff
@@ -51,18 +72,35 @@ export default class QuanticSummary extends LightningElement {
   }
 
   get range() {
-    return ` ${this.state.firstResult} - ${this.state.lastResult}`;
+    return `${this.state.firstResult}-${this.state.lastResult}`;
```

Comment:
Isn't the text cramped without the whitespaces?

Reply:
This is according to the mock ups where I think it looks all right. You disagree? 
<img width="513" alt="Screen Shot 2021-07-22 at 10 53 33 AM" src="https://user-images.githubusercontent.com/16785453/126660378-85ec27c6-c335-43b1-b573-614ac9d42ea7.png">